<launch>
<node pkg="mobot6_interface" type="twins" name="socket" respawn="true" launch-prefix="xterm -e" />
<node pkg="mobot6_interface" type="socket_send" name="socket_send" respawn="true" launch-prefix="xterm -e" />
</launch>
